modules/apps/batch-engine/batch-engine-api/src/main/java/com/liferay/batch/engine/action/ImportTaskPreAction.java

```diff
@@ -15,7 +16,8 @@ public interface ImportTaskPreAction {
 
 	public void run(
 			BatchEngineImportTask batchEngineImportTask,
-			ImportTaskContext importTaskContext, Object item)
+			ImportTaskContext importTaskContext, Object item,
+			UnsafeFunction<?, ?, Exception> unsafeFunction)
```

Comment:
I think it would be easier for now and also for the future to have the `BatchEngineTaskItemDelegate` that is executing the import task instead. Something like:

```java
public void run(
	BatchEngineImportTask batchEngineImportTask,
	BatchEngineTaskItemDelegate<?> batchEngineTaskItemDelegate,
	ImportTaskContext importTaskContext, Object item)
```